src/h265_sps_parser.cc

```diff
@@ -673,21 +672,17 @@ uint32_t H265SpsParser::SpsState::getPicSizeInCtbsY() {
   uint32_t CtbLog2SizeY = MinCbLog2SizeY +
       log2_diff_max_min_luma_coding_block_size;
   // Equation (7-12)
-  uint32_t MinCbSizeY = 1 << MinCbLog2SizeY;
```

Comment:
Do you really need to delete these 4 lines to compile? They should be useful at some point. 

If you really need, please just comment them out